--- a/osdep/LinuxEthernetTap.cpp	2022-06-28 05:13:57.000000000 +0800
+++ b/osdep/LinuxEthernetTap.cpp	2022-10-31 14:15:42.626923438 +0800
@@ -273,11 +273,11 @@
 		if (!isOldLinuxKernel()) {
 			ifr.ifr_ifru.ifru_hwaddr.sa_family = ARPHRD_ETHER;
 			_mac.copyTo(ifr.ifr_ifru.ifru_hwaddr.sa_data,6);
-			if (ioctl(sock,SIOCSIFHWADDR,(void *)&ifr) < 0) {
-				::close(sock);
-				printf("WARNING: ioctl() failed setting up Linux tap device (set MAC)\n");
-				return;
-			}
+			// if (ioctl(sock,SIOCSIFHWADDR,(void *)&ifr) < 0) {
+			// 	::close(sock);
+			// 	printf("WARNING: ioctl() failed setting up Linux tap device (set MAC)\n");
+			// 	return;
+			// }
 
 			ifr.ifr_ifru.ifru_mtu = (int)_mtu;
 			if (ioctl(sock,SIOCSIFMTU,(void *)&ifr) < 0) {
--- a/osdep/OSUtils.cpp	2022-06-28 05:13:57.000000000 +0800
+++ b/osdep/OSUtils.cpp	2022-10-31 14:18:25.549673365 +0800
@@ -417,7 +417,7 @@
 	return std::string("/var/db/zerotier-one");
 #else
 	// Use /var/lib for Linux and other *nix
-	return std::string("/var/lib/zerotier-one");
+	return std::string("/etc/storage/zerotier-one");
 #endif
 
 #endif
